src/components/books.js

```diff
@@ -0,0 +1,32 @@
+import { useState } from 'react';
+import { v4 } from 'uuid';
+import Book from './book';
+
+const Books = () => {
+  const [items, setItems] = useState([]);
+  const handleDelete = (itemId) => setItems(items.filter((item) => item.id !== itemId));
+  const handleSubmit = (event) => {
+    event.preventDefault();
+    const form = event.target;
+    if (form.title.value !== '' && form.author.value !== '') {
+      const book = { id: v4(), title: form.title.value, author: form.author.value };
+      setItems([...items, book]);
+      form.reset();
+    }
+  };
+
+  return (
+    <section>
+      <h2>Books List</h2>
+      <ul>{items.map((item) => <Book key={item.id} book={item} handler={handleDelete} />)}</ul>
```

Comment:
Great job @mahammad-mostafa 🤝

- [ ] As stated in the project requirements, kindly ensure the books component renders both the booklist and the book components properly. The book component should contain the delete button which is not rendered in the UI.
![book-list](https://github.com/mahammad-mostafa/book-store-app/assets/63917725/99e4b1a1-9c3d-4eda-ab69-626e0436195f)
- Kindly use the project requirements below
![book](https://github.com/mahammad-mostafa/book-store-app/assets/63917725/4f30008f-cd94-47df-9497-b19eff1763f1)

Reply:
Thanks @1GORDON for feedback, But I understood that the button should be added with the `Book` component item. That is why it appears whenever you add a book to the list.

Reply:
Thank you for your response @mahammad-mostafa. However, at the moment the buttons are not functional. We only need the UI to be rendered properly. At the moment we won't be adding and deleting books.